### ISSUE TITLE
Locked and unavailable mount type  look the same

### ISSUE THREAD

**Issue by IsaSih** [rancher-sandbox/rancher-desktop#5449]
### Problem Description

Once mount type is locked in a macOS version older than Ventura 13.0, the appearance of the radio buttons are the same, making it confusing for the user to distinguish between what field is locked and what is unavailable 
![lock-mount-type](https://github.com/rancher-sandbox/rancher-desktop/assets/70187881/b57f777f-e4ca-4e37-86ff-fffaf3f30c05)

### Proposed Solution

Place a white lock icon right next to each option instead of the whole component.

**Comment by s0nea** [comment]
Hi @IsaSih,

Thank you for testing and opening the issue.

I see your point. However the current UI corresponds to the planned design. I'll discuss it with the UI team and see if we can improve it.

**Comment by jandubois** [comment]
I'm removing the milestone, as this behaviour (the lock icon is after the setting name, not behind every setting value) is by design.

We need to discuss this in the UI/UX meeting before we can change this.

**Comment by jandubois** [comment]
In some cases we have competing hover-tips: One explains that virtiosfs requires VZ emulation. The other should be that the setting cannot be changed because it is locked.

I think the locked tip should always take precedence when the setting is locked (also subject to UI/UX discussion).

**Comment by s0nea** [comment]
> I'm removing the milestone, as this behaviour (the lock icon is after the setting name, not behind every setting value) is by design.

![242025877-52db699d-db23-4698-b2a7-4bcee5f9534b](https://github.com/rancher-sandbox/rancher-desktop/assets/8761082/14c46389-913b-4bf4-82fd-650d9372e124)

As you can see on the screen there is a differentiation between locked and disabled + locked (the radio itself is filled / unfilled).

> 
> We need to discuss this in the UI/UX meeting before we can change this.

Yes :+1: 

> In some cases we have competing hover-tips: One explains that virtiosfs requires VZ emulation. The other should be that the setting cannot be changed because it is locked.

The hint about the setting being locked will only appear on the lock icon itself. It will never appear on the actual radio button.

The radio button itself will only show information why it's disabled and the experimental icon the "Experimental" status.

> I think the locked tip should always take precedence when the setting is locked (also subject to UI/UX discussion).

Yes :+1:

**Comment by s0nea** [comment]
> * for unavailable settings (virtiofs in the video) the button should have a grey background, as shown [above](https://github.com/rancher-sandbox/rancher-desktop/issues/5449#issuecomment-1708418471)

-> Created a separate issue here: https://github.com/rancher-sandbox/rancher-desktop/issues/5527
